wordpress-mobile/WordPress-iOS
# ULS: Add visual blur to Prologue button view

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/401
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/487

This uses the Auth changes to show the visual blur behind the Prologue button view.

To test:
- Verify the Prologue button view background is a bit blurry. 😄 

| Before | After |
|--------|-------|
| ![before_light](https://user-images.githubusercontent.com/1816888/94306756-c3665c80-ff30-11ea-9c2c-af3502e29ba8.png) | ![after_light](https://user-images.githubusercontent.com/1816888/94306771-c9f4d400-ff30-11ea-9ee9-4609073bccae.png) |
| ![before_dark](https://user-images.githubusercontent.com/1816888/94306787-d11be200-ff30-11ea-8efe-ffbeabdb5370.png) | ![after_dark](https://user-images.githubusercontent.com/1816888/94306800-d7aa5980-ff30-11ea-90d2-4213ac8cc480.png) |

PR submission checklist:
